protocol/triple/health/triple_health/health.proto

```diff
@@ -20,7 +20,7 @@
 
 syntax = "proto3";
 
-package grpc.health.v1;
+package dubbo.health.v1;
```

Comment:
In dubbo3, it use dubbogo. Does it affect compatibility?

Reply:
We just need to make sure we don't have the same proto file throughout the project